Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:latest
+
+RUN apt-get update
+RUN apt-get upgrade -y
+
+RUN apt-get install curl -y
+RUN apt-get install build-essential -y
+RUN apt-get install libssl-dev -y
+RUN apt-get install pkg-config -y
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain stable -y
+
+RUN mkdir session-open-group-server
+ADD . /session-open-group-server
+WORKDIR session-open-group-server
+
+RUN ~/.cargo/bin/cargo build --release
+RUN mkdir ./target/release/data
+WORKDIR ./target/release/data
+
+RUN openssl genpkey -algorithm x25519 -out x25519_private_key.pem
+RUN openssl pkey -in x25519_private_key.pem -pubout -out x25519_public_key.pem
```

Comment:
not sure how persistent storage works on docker, but if we change this file and as the generation of the keys is that late in the building process, won't it generate a new public/private key on each change? 

If yes, should we add a guard to not overwrite those file, or run this step right after the `FROM debian:latest` ?

Reply:
It's expected behavior. The keys will be stored in the directory `/session-open-group-server/target/release/data` which, if the user is following the instructions in the `DOCKER.md` file, is already mounted as a volume, alongside all the group data which is supposed to persist.

This is how it works:
1. User fectes pre-built image/compiles its own: directory `/session-open-group-server/target/release/data` is empty
2. User starts image: data from volume `session-open-group-data` is mounted into dir `/session-open-group-server/target/release/data`.
    - IF the volume is empty (first time running the container): the keys generated during build are used normally
    - IF the volume isn't empty: all the data is mounted from the volume, keys included, overwriting the ones generated on image build